python/orca/example/learn/pytorch/brainMRI/README.md

```diff
@@ -0,0 +1,55 @@
+## Trouble Shooting
+- [x] no support of val set?
+- [ ] Difficult to identify a customized loss function; for example, dice coefficient
```

Comment:
I think bce_dice_loss can directly pass to the estimator? (at least for torch_distributed)